zeebe/benchmarks/project/src/main/java/io/camunda/zeebe/OperateClient.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.camunda.zeebe.util.CloseableSilently;
+import java.io.IOException;
+import java.net.Authenticator;
+import java.net.PasswordAuthentication;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpClient.Version;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.time.Duration;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executor;
+import org.agrona.CloseHelper;
+
+final class OperateClient implements CloseableSilently {
+  private static final ObjectMapper MAPPER = new ObjectMapper();
+
+  private final HttpClient client;
+
+  private final URI baseUri;
+
+  OperateClient(final URI baseUri, final Executor executor) {
+    this.baseUri = baseUri;
+
+    client =
+        HttpClient.newBuilder()
+            .authenticator(
+                new Authenticator() {
+                  @Override
+                  protected PasswordAuthentication getPasswordAuthentication() {
+                    return new PasswordAuthentication("demo", "demo".toCharArray());
```

Comment:
:thought_balloon: Only supports basic auth with default credentials. If we need more, we can build it then :shrug:

Reply:
I guess you tested it? I think for 8.6/8.7 I remember something related you need to login and safe the cookie or something